src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -55,14 +57,20 @@
   <!-- Filters. -->
   <filter-bar state="state"></filter-bar>
 
-  <div ng-show="showEditHelpText()" class="text-center start-screen">
+  <div ng-show="getShowEditHelpText()" class="text-center start-screen">
     <h2>This dashboard is empty. Let's fill it up!</h2>
     <p>Click the <a class="btn btn-xs navbtn-inverse" ng-click="kbnTopNav.open('add'); toggleAddVisualization = !toggleAddVisualization" aria-label="Add visualization">Add</a> button in the menu bar above to add a visualization to the dashboard. <br/>If you haven't setup a visualization yet visit <a href="#/visualize" title="Visualize">"Visualize"</a> to create your first visualization.</p>
   </div>
 
+  <div ng-show="getShowViewHelpText()" class="text-center start-screen">
```

Comment:
mm, very good point, it does sound like it's returning the actual text.

re: `isViewHelpTextVisible`
> "If the edit help text is visible, render this part of the template."

The edit help text *is* this part of the template. so in my mind that statement is:
`"If the edit help text is visible, make the edit help text visible."`
That function isn't checking if the help text is visible, it's checking to see if it *should* be visible. Otherwise it sounds circular to me.

I'm looking at existing places in our code base that use ng-show and it looks like we generally use `ng-show=showX` but those are likely variables, not functions.  There are a few places we use something to the effect of `isShown`, but on closer inspection, it's looking to see if another part of the ui is shown, not itself.  E.g.:

```
        <span ng-show="isCategoryAxisAdvancedOptionsOpen">
          Hide
        </span>
```
It's showing that portion of the UI if another part of the UI is already shown, not if it itself is shown. 
Am I making sense?